react-app/src/features/welcome/index.tsx

```diff
@@ -38,83 +39,87 @@ export const Welcome = () => {
             <img src={OneMacLogo} alt="onemac" className="p-4 min-w-[400px]" />
           </h1>
           <p className="text-center text-white/100 font-light text-xl font-sans">
-            Welcome to the official submission system for paper-based state plan
-            amendments (SPAs) and section 1915 waivers.
+            Welcome to the official submission system for paper-based state plan amendments (SPAs)
+            and section 1915 waivers.
           </p>
         </div>
       </div>
       {/* End Hero Section */}
       {/* Two Column Main Layout */}
       <div className="max-w-screen-xl mx-auto p-4 lg:px-8">
+        <div className="flex flex-col justify-center items-center">
```

Comment:
We could move the `max-width` from the card and just use the ol `margin: auto` trick to center it. Would remove the need for this extra `div`. Something like-
```
<div class="m-auto max-w-[767px]">
  <h2 class="text-2xl font-bold">New and Notable</h2>
  <CardWithTopBorder>
  ...
  </CardWithTopBorder>
</div>
```

Reply:
Love this! Great suggestion, thank you 🎉